### PR TITLE
Add ETH-B keeper

### PR DESCRIPTION
**Pull request by grandizzy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,11 +65,13 @@ After following the setup procedure below, this keeper works out of the box unde
...
+    - `FLIP_MINIMUM_ETH_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_ETH_B_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_BAT_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_USDC_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_USDC_B_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_WBTC_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_TUSD_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_KNC_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_ZRX_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_MANA_A_AUCTION_ID_TO_CHECK` | `FLIP_MINIMUM_USDT_A_AUCTION_ID_TO_CHECK`: Recommendation under introduction section
+    - `FLIP_ETH_DISCOUNT` | `FLIP_ETH_B_DISCOUNT` | `FLIP_BAT_A_DISCOUNT` | `FLIP_USDC_A_DISCOUNT` | `FLIP_USDC_A_DISCOUNT` | `FLIP_WBTC_A_DISCOUNT` | `FLIP_TUSD_A_DISCOUNT` | `FLIP_KNC_A_DISCOUNT` | `FLIP_ZRX_A_DISCOUNT` | `FLIP_MANA_A_DISCOUNT` | `FLIP_USDT_A_DISCOUNT` | `FLIP_PAXUSD_A_DISCOUNT`: Discount from ETH's or BAT's or USDC's FMV or WBTC's FMV, which will be used as the bid price
```
Comment (EdNoepel) [inline README.md:74]: Wouldn't it still be `FLIP_ETH_A_DISCOUNT` instead of `FLIP_ETH_DISCOUNT`?

**Review by EdNoepel — COMMENTED** [review]
One comment inline about the README.  I also think the README could be simplified to use examples rather than updating everything every time a new collateral comes out.  It'd reduce doc maintenance and be easier to read.  Something to consider for next time.

File: README.md (hunk 1 above)
Comment (grandizzy) [reply README.md:74]: there wasn't a `FLIP_ETH_A_DISCOUNT` defined, I updated README to get inline with env vars: https://github.com/makerdao/dockerized-auction-keeper/blob/master/env/environment.sh#L29

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**